data_validation/config_manager.py

```diff
@@ -386,7 +390,9 @@ def get_source_ibis_table_from_query(self):
     def get_source_ibis_calculated_table(self, depth=None):
         """Return mutated IbisTable from source
         n: Int the depth of subquery requested"""
-        if self.validation_type == consts.CUSTOM_QUERY:
+        if (
```

Comment:
That was due to the comment there black was inserting the parantheses. The comment is no longer necessary since the code itself says that. Removing the comment - removed the ()